docs/section21/Rule_21-14.md

```diff
@@ -0,0 +1,37 @@
+
+# Boiler - Rule 21-14  
+
+**Rule ID:** 21-14  
+**Rule Description:** When the baseline building is modeled with a hot water plant, served by purchased HW system, hot water supply temperature reset is not modeled.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased hot water or steam.
+
+**Manual Check:** Yes  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** 
+
+1. check_purchased_chw_hhw()
+
+**Applicability Checks:**
+
+- Check if B-RMR is modeled with purchased chilled water or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(B_RMR)`
+
```

Comment:
this doesnt need to check purchased chilled water, correct?

Reply:
right.  updated